src/service/standard-actions-impl.js

```diff
@@ -96,6 +99,34 @@ export class StandardActions {
       }
     });
   }
+
+  /**
+   * Handles "show" action. This is a very simple action where "display: none"
+   * is removed from the target element.
+   * @param {!./action-impl.ActionInvocation} invocation
+   */
+  handleShow(invocation) {
+    const target = dev().assertElement(invocation.target);
+    this.resources_.deferMutate(target, () => {
+      if (target.classList.contains('i-amphtml-element')) {
+        target./*OK*/expand();
+      } else {
+        toggle(target, true);
+      }
+    });
+  }
+
+  /**
+   * Handles "toggle" action.
+   * @param {!./action-impl.ActionInvocation} invocation
+   */
+  handleToggle(invocation) {
+    if (getStyle(dev().assertElement(invocation.target), 'display') == 'none') {
```

Comment:
we should use `getComputed` style here so if element is hidden some other way (e.g. author defined CSS class) we toggle properly.